Remove snapshot controller check in CI and deployment scripts

humblec opened the pull request:
The version of kube server has been imposed on this script to check it
is < v1.17  which is no longer required.

- [x] snapshot controller installation scripts are changed


Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on scripts/install-snapshot.sh:
```diff
@@ -99,14 +99,6 @@ if ! get_kube_version=$(kubectl version --short) ||
 fi
 
 KUBE_VERSION=$(echo "${get_kube_version}" | grep "^Server Version" | cut -d' ' -f3)
```
parsing kubeversion code is also not required anymore?

Madhu-1 commented on scripts/travis-functest.sh:
```diff
@@ -40,8 +40,6 @@ fi
 # functional tests
 make run-e2e E2E_ARGS="${*}"
 
-if [[ "${KUBE_MAJOR}" -ge 1 ]] && [[ "${KUBE_MINOR}" -ge 17 ]]; then
```
I still see the code related to snapshot <1.17 in line 33

Madhu-1 commented on scripts/travis-helmtest.sh:
```diff
@@ -35,28 +35,21 @@ sudo scripts/minikube.sh k8s-sidecar
-    scripts/install-snapshot.sh install
-fi
+# delete snapshot CRD created by ceph-csi in rook
+scripts/install-snapshot.sh delete-crd
```
this is not required anymore?

Madhu-1 commented on scripts/travis-helmtest.sh:
```diff
@@ -35,28 +35,21 @@ sudo scripts/minikube.sh k8s-sidecar
-    scripts/install-snapshot.sh cleanup
-fi
+# cleanup
+# delete snapshot CRD
```
need to remove this comment

Madhu-1 commented on scripts/travis-helmtest.sh:
```diff
@@ -35,28 +35,21 @@ sudo scripts/minikube.sh k8s-sidecar
-fi
+# delete snapshot CRD created by ceph-csi in rook
+scripts/install-snapshot.sh delete-crd
+# install snapshot controller
```
install snapshot controller and create snapshot CRD

humblec commented:
/retest all

nixpanic commented:
> /retest all

Why do you run this? It burns CI resources for no obvious reason :sob:

nixpanic commented on scripts/install-snapshot.sh:
```diff
@@ -84,30 +84,6 @@ function delete_snapshot_crd() {
     kubectl delete -f "${VOLUME_SNAPSHOT}" --ignore-not-found
 }
 
```
Typo in the commit message, not version 1.7 but 1.17.

humblec replied:
corrected the same. thanks :+1:

nixpanic commented on scripts/travis-functest.sh:
```diff
@@ -27,21 +27,12 @@ sudo scripts/minikube.sh create-block-pool
-    scripts/install-snapshot.sh install
-fi
+# install snapshot controller and create snapshot CRD
+scripts/install-snapshot.sh install
```
`scripts/install-snapshot.sh delete-crd` is not called anymore, can it be removed from the script?

humblec replied:
yeah that can be removed, done

humblec committed:
deploy: remove the snapshot controller installation check

no need to have 1.17 kube version check anymore  before we install
snapshot controller.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec committed:
ci: remove check for snapshot controller installation and cleanup

At present, the check is performed to validate the version of kube
is v1.17 and this commit remove the same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented:
@Mergifyio rebase

humblec commented:
@nixpanic ptal.. thanks.

humblec commented:
@Mergifyio rebase
